app/src/nvdata.h

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ *   (c) 2018 -2024 Zondax AG
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ********************************************************************************/
+ #pragma once
+
+#include "coin.h"
+#include "constants.h"
+#include "zxerror.h"
+#include "zxmacros.h"
+#include <stdbool.h>
+
+#define SPEND_LIST_SIZE 5
```

Comment:
Are we able to increase this value? Do we have any restriction in terms of memory/storage?